internal/store/logs.go

```diff
@@ -82,7 +83,10 @@ func (s *Store) Collect(groupID string, newLog *opsmngr.LogCollectionJob) (*opsm
 func (s *Store) DownloadLog(groupID, host, name string, out io.Writer, opts *atlas.DateRangetOptions) error {
 	switch s.service {
 	case config.CloudService, config.CloudGovService:
-		_, err := s.client.(*atlas.Client).Logs.Get(s.ctx, groupID, host, name, out, opts)
+		start, _ := strconv.ParseInt(opts.StartDate, 10, 64)
+		end, _ := strconv.ParseInt(opts.EndDate, 10, 64)
+		logs, _, err := s.clientv2.MonitoringAndLogsApi.GetHostLogs(s.ctx, groupID, host, name).StartDate(start).EndDate(end).Execute()
```

Comment:
@wtrocki This is Failing due to:

GET /api/atlas/v2/groups/6421987cdc060f20518edb62/clusters/atlas-eivyf7-shard-00-00.0y9t6.mongodb-dev.net:27017/logs/mongodb.gz.gz?endDate=0&startDate=0 HTTP/1.1
H
and

{"detail":"Unexpected error.","error":500,"errorCode":"UNEXPECTED_ERROR","parameters":[],"reason":"Internal Server Error"}
Error: undefined response type

Command: bin/atlas logs download atlas-eivyf7-shard-00-00.0y9t6.mongodb-dev.net:27017 mongodb.gz --projectId 6421987cdc060f20518edb62 --debug